compare50/_data.py

```diff
@@ -125,6 +125,10 @@ class Submission:
     def __attrs_post_init__(self):
         object.__setattr__(self, "files", tuple(
             [File(pathlib.Path(path), self) for path in self.files]))
+        object.__setattr__(self, "large_files", tuple(
```

Comment:
We should really use `converter` for this. I don't think we actually need them to be Paths rather than strings, but if so, you can just add `converter=lambda vals: tuple(map(pathlib.Path, vals))` to both large_files and undecodable_files

Reply:
If strings are fine, then the existing `converter=tuple` that i added will work.

Reply:
Ah right, because we don't need `self` here. I vote we stick to `pathlib.Path` because we use that everywhere else.